translate/storage/test_pypo.py

```diff
@@ -238,17 +236,14 @@ def test_output_str_unicode(self):
         assert len(pofile.units) == 1
         print(bytes(pofile))
         thepo = pofile.units[0]
-        # On Python 2, str() returns bytestrings while on Python 3, str() returns unicode
-        assert str(thepo) == posource if six.PY3 else posource.encode("UTF-8")
-        assert six.text_type(thepo) == posource
+        assert str(thepo) == posource
+        assert str(thepo) == posource
```

Comment:
One of these assertions can be removed now.

Reply:
Removed. I really didn't look at most of the code, so it was easy to miss such cases.